### DISCUSSION TITLE
is there kustomize version used by flux kustomization?

### DISCUSSION THREAD

**Discussion by maplewf** [fluxcd/flux2#4518] [General] [answered]
Hi,
I'm new in flux, I wonder what kind of kustomize version used by flux kustomization to do the build? where I can find this information? or I'm totally wrong, there is no such thing? thanks for help

**Comment by stefanprodan** [comment, marked as answer]
The current version compatible with Flux kustomize-controller is Kustomize v5.3.0. We mention this in the changelog, every time we bump a dependency.

For example, in [kustomize-controller changelog](https://github.com/fluxcd/kustomize-controller/blob/main/CHANGELOG.md) you'll see:

```
Update Kustomize to v5.3.0
```

The Flux controllers don't use any external binaries, instead we use the Go SDKs of Kustomize, Helm, Git, etc. The version of each SDK can be found in the `go.mod` file of each controller.

If you look in the [kustomize-controller go.mod](https://github.com/fluxcd/kustomize-controller/blob/main/go.mod), you'll see the version there, for example:

```go
// Pin kustomize to v5.3.0
replace (
	sigs.k8s.io/kustomize/api => sigs.k8s.io/kustomize/api v0.16.0
	sigs.k8s.io/kustomize/kyaml => sigs.k8s.io/kustomize/kyaml v0.16.0
)
```

**Reply by maplewf** [reply]
This is very clear, thanks very much for your answer. Though I think it would be more convenient if flux command line can reveal this info

**Reply by stefanprodan** [reply]
Flux integrates with dozens of SDKs, I’m not considering listening them in the CLI.
